Redesign front page (#907)
docat-org/docat: README.md

[![build](https://github.com/docat-org/docat/workflows/docat%20ci/badge.svg)](https://github.com/docat-org/docat/actions)
[![Gitter](https://badges.gitter.im/docat-docs-hosting/community.svg)](https://gitter.im/docat-docs-hosting/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

> ## Why DoCat?

Reviewer: Is `DoCat` really the casing? Haven't seen it anywhere else ...

> When generating static developer documentation using

Reviewer: ```suggestion
When generating static documentation using
```

[mkdocs](https://www.mkdocs.org/), [sphinx](http://www.sphinx-doc.org/en/master/), ...
hosting just one version of the docs might not be enough.
Many users might still use older versions and might need to read
those versions of the documentation.

> Docat solves this problem by providing a simple tool that can

Reviewer: ```suggestion
Docat solves this problem by providing a simple tool that
```

> host multiple documentation projects with multiple versions.

Reviewer: ```suggestion
hosts multiple documentation projects with multiple versions.
```

> *The main design decision with docat was to keep the tool as simpel as possible*

Reviewer: ```suggestion
*The main design decision with docat was to keep the tool as simple as possible.*
```

for all `POST`/`PUT` and `DELETE` http calls.

<details>
  <summary>docat http basic authentication example</summary>

This example shows how to configure the NGINX inside the docker image
to be password protected using http basic auth.

1) Create your `htpasswd` file.
2) And a custom `default.conf` NGINX config:

  ```
  upstream python_backend {
      server 127.0.0.1:5000;
  }

  server {
      listen 80 default_server;
      listen [::]:80 default_server;

      auth_basic           "Administrator’s Area";
      auth_basic_user_file htpasswd;

      root /var/www/html;

      # Add index.php to the list if you are using PHP
      index index.html index.htm index.nginx-debian.html;

      server_name _;

      location /doc {
          root /var/docat;
          autoindex on;
          autoindex_format json;
      }

      location /api {
          client_max_body_size 100M;
          proxy_pass http://python_backend;
      }

      location / {
      }
  }
  ```

3) Mounted to the correct location inside the container:

  ```
  docker run \
    --detach \
    --volume $PWD/docat-run:/var/docat/ \

>     --volume $PWD/nginx/default.conf:/etc/nginx/http.d/default.conf \

Reviewer 2: ```suggestion
    --volume $PWD/nginx/default.conf:/app/docat/docat/nginx/default \
```

since the env substitution

Go to [localhost:8000](http://localhost:8000) to view your docat instance:

<img src="doc/assets/docat.gif" width="100%" />

### Using DoCat

> > 🛈 Please note that docat does not provide any way to write documentation

Reviewer: ```suggestion
> 🛈 Please note that docat does not provide any way to write documentation.
```

> > the tool only hosts documentation.

Reviewer: ```suggestion
> It's sole responsibility is to host documentation.
```

>

> > There are many awesome tools to write developer documenation:

Reviewer: ```suggestion
> There are many awesome tools to write documenation:
```

> > [mkdocs](https://www.mkdocs.org/), [sphinx](http://www.sphinx-doc.org/en/master/),
> > [mdbook](https://rust-lang.github.io/mdBook/) ...

Reviewer: ```suggestion
> - [mkdocs](https://www.mkdocs.org/)
> - [sphinx](http://www.sphinx-doc.org/en/master/)
> - [mdbook](https://rust-lang.github.io/mdBook/)
> - ...
```

> A small tool called [docatl](https://github.com/docat-org/docatl) is provided

Reviewer: ```suggestion
A CLI tool called [docatl](https://github.com/docat-org/docatl) is available
```

for easy interaction with the docat server.
However, interacting with docat can also be done through [`curl`](doc/getting-started.md).

> So in order to push documentation (and tag as `latest`) in the folder `docs/` simply run:

Reviewer: ```suggestion
To push documentation (and tag as `latest`) in the folder `docs/` simply run:
```

Go to [localhost:8000](http://localhost:8000) to view your docat instance:

> <img src="doc/assets/docat.gif" width="100%" />

Reviewer 2: love it
